src/Text/Pandoc/Writers/LaTeX.hs

```diff
@@ -1454,15 +1454,35 @@ citationsToBiblatex
                   NormalCitation -> "autocite"
 
 citationsToBiblatex (c:cs)
-  | all (\cit -> null (citationPrefix cit) && null (citationSuffix cit)) (c:cs)
+  | all (\cit -> null (citationPrefix cit) && null (citationSuffix cit)) 
+    (c:cs)
     = do
       let cmd = case citationMode c of
                     SuppressAuthor -> "\\autocite*"
                     AuthorInText   -> "\\textcite"
                     NormalCitation -> "\\autocite"
       return $ text cmd <>
                braces (literal (T.intercalate "," (map citationId (c:cs))))
-  | otherwise = do
+  | all (\cit -> null (citationSuffix cit)) (c:cs) 
+    = do  
+      let cmd = case citationMode c of 
+                    SuppressAuthor -> "\\autocites*"
+                    AuthorInText   -> "\\textcites"
+                    NormalCitation -> "\\autocites"
+
+      let pfxs = tail $ scanl (\prev cit -> let pfx = citationPrefix cit in
+                if pfx == [] then prev else pfx) [] (c:cs)
+
+      let pfxGroups = map (map snd) $ groupBy (\(pre, _) (pre1, _) -> pre == pre1)
+            (zip pfxs (c:cs))  
+      prefixes <- mapM (\group -> 
+                (fmap brackets) $ inlineListToLaTeX $ citationPrefix $ head group)
```

Comment:
`head` is also a partial function.
`fmap brackets $ xxx` is more idiomatically written `brackets <$> xxx`.